### ISSUE TITLE
JSON Pretty Preview Parse Error

### ISSUE THREAD

**Issue by qwo** [a85/POSTMan-Chrome-Extension#1777]
I was on a version pre 4.0 and this seemed to work fine 
error message: `Something went wrong while parsing the response as JSON: Unexpected ')'`
I am now on 

```
Version 4.0.3 (4.0.3)
OSX  El Capitan  
Desktop Application
```

<img width="961" alt="screenshot 2016-03-01 16 10 21" src="https://cloud.githubusercontent.com/assets/3683993/13441994/27661032-dfc8-11e5-840c-25ea848ee362.png">

the raw json is as such, i sanitized my actual json input but the gist is. some APIs lead with `)]}',` for security. It looks 

```
)]}',
{"foo": "bar"}
```

**Comment by abhijitkane** [comment]
Thanks for reporting this. We've fixed this in our internal build and will push out the new version soon.

**Comment by SamvelRaja** [comment]
This had been fixed in the latest release. Can you update the app?  Latest version: `4.1.0`

**Comment by qwo** [comment]
Looks like it was fixed in 4.0.11. Thanks
